logging_test.go

```diff
@@ -53,6 +53,7 @@ func TestLogging(t *testing.T) {
 			Case: func(t *testing.T, cache LoggingCache) {
 				assert.NoError(t, cache.Put("id", &options.CachedLogger{ID: "id"}))
 				assert.Equal(t, 1, cache.Len())
+				time.Sleep(time.Second)
```

Comment:
Just use the change in my PR (or approve it and merge): https://github.com/mongodb/jasper/pull/170